e2e/fixtures/authenticatedUserPages.ts

```diff
@@ -68,10 +69,18 @@ export const testAuth = base.extend<
       })
 
       for (const user of Object.values(userConfigs)) {
-        await base.step(`create test user - ${user.username}`, async () => {
-          const userId = await createTestUser(user, getAdminPAT(), page)
-          expect(userId).not.toBeUndefined()
-          userIds.push(userId)
+        const userId = await base.step(
+          `create test user - ${user.username}`,
+          async () => {
+            const userId = await createTestUser(user, getAdminPAT(), page)
+            expect(userId).not.toBeUndefined()
+            userIds.push(userId)
+            return userId
+          },
+        )
+
+        await base.step(`certify test user - ${user.username}`, async () => {
+          await setCertifiedUserStatus(userId, true, getAdminPAT(), page)
```

Comment:
Users must be certified to upload a file -- certify new users by default, similar to [backend integration tests](https://github.com/Sage-Bionetworks/Synapse-Repository-Services/blob/e84f3c2556c4458348c27cd11edfad82ceed788e/integration-test/src/test/java/org/sagebionetworks/SynapseClientHelper.java#L69C18-L69C18)